openmetadata-ui/src/main/resources/ui/src/pages/AddIngestionPage/AddIngestionPage.component.tsx

```diff
@@ -97,6 +98,8 @@ const AddIngestionPage = () => {
                     jsonData['api-error-messages']['deploy-ingestion-error']
                   );
                 } else {
+                  reject();
```

Comment:
Do we need to reject here if we are already rejecting in the catch block?

Reply:
Ya correct.! I ll make changes